environment-variables/validate-environment-variables/snippet.go

```diff
@@ -0,0 +1,28 @@
+package snippets
+
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+// See https://godoc.org/github.com/kelseyhightower/envconfig
```

Comment:
would you mind adding a README.md to this snippet's directory with a couple of lines explaining the backstory about why you would want to do this?

Reply:
:+1: